jsonschema-core.xml

```diff
@@ -800,49 +800,302 @@
+                    is RECOMMENDED that implementations detect these cases and raise
+                    an error when they occur.
+                </t>
+                <t>
+                    Meta-schemas that do not use "$vocabulary" MUST be considered to
+                    require the Core vocabulary just as if "$vocabulary" were used
+                    with only the Core vocabulary present with a value of true.
+                </t>
+                <t>
+                    The current URI for the Core vocabulary is:
+                    <eref target="https://json-schema.org/draft-08/vocabularies/core"/>.
+                </t>
+                <t>
+                    The current URI for the corresponding meta-schema is:
+                    <eref target="https://json-schema.org/draft-08/core"/>.
+                </t>
+            </section>
+            <section title="Example Meta-Schema With Vocabulary Declarations"
+                     anchor="example-meta-schema">
+                <figure>
+                    <preamble>
+                        This meta-schema explicitly declares both the Core and Applicator
+                        vocabularies, and combines their meta-schemas with an "allOf".
+                        It additionally restricts the usage of the Applicator vocabulary
+                        by forbidding the keyword prefixed with "unevaluated".  It also
+                        describes a keyword, "localKeyord", that is not part of either
+                        vocabulary.  Note that it is its own meta-schema,
+                        as it relies on both the Core vocabulary (as all schemas do)
+                        and the Applicator vocabulary (for "allOf"0.
+                    </preamble>
+                    <artwork>
+<![CDATA[
+{
+  "$schema": "https://json-schema.org/draft-08/core-app-example#",
+  "$id": "https://json-schema.org/draft-08/core-app-example",
+  "$recursiveAnchor": true,
+  "$vocabulary": {
+    "https://json-schema.org/draft-08/vocabularies/core": true,
+    "https://json-schema.org/draft-08/vocabularies/applicator": true
+  },
+  "allOf": [
+    {"$ref": "https://json-schema.org/draft-08/core"},
+    {"$ref": "https://json-schema.org/draft-08/applicator"}
+  ],
+  "patternProperties": {
+    "^unevaluated.*$": false
+  },
+  "properties": {
+    "$comment": "Not in vocabulary, but validated if used",
+    "localKeyword": {
+      "type": "string"
+    }
+  }
+}
+]]>
+                    </artwork>
+                    <postamble>
+                        As shown above, even though each of the referenced standard
+                        meta-schemas declares its corresponding vocabulary, this new
+                        meta-schema must re-declare them for itself.  It would be
+                        valid to leave the core vocabulary out of the "$vocabulary" keyword,
+                        but it MUST be referenced through the "allOf" keyword in order
```

Comment:
```suggestion
                        but it has to be referenced through the "allOf" keyword in order
```
Using MUST regarding an example does not quite seem a requirement.

Reply:
or perhaps "must" not in caps...